src/Tester/StreamingTests/PullingAgentManagementTests.cs

```diff
@@ -16,30 +16,23 @@ namespace UnitTests.StreamingTests
     [DeploymentItem("ClientConfigurationForStreamTesting.xml")]
     [DeploymentItem("OrleansProviders.dll")]
     [TestClass]
-    public class PullingAgentManagementTests : UnitTestSiloHost
+    public class PullingAgentManagementTests : UnitTestSiloHostPerFixture
```

Comment:
Does the word `Fixture` here have the same semantic meaning as it does in the xUnit test framework?
http://xunit.github.io/docs/shared-context.html